ghost/admin/lib/koenig-editor/addon/components/koenig-card-image.js

```diff
@@ -5,13 +5,37 @@ import {
     IMAGE_MIME_TYPES
 } from 'ghost-admin/components/gh-image-uploader';
 import {action, computed, set, setProperties} from '@ember/object';
+import {fetch} from 'fetch';
 import {utils as ghostHelperUtils} from '@tryghost/helpers';
 import {isEmpty} from '@ember/utils';
 import {run} from '@ember/runloop';
 import {inject as service} from '@ember/service';
 
 const {countWords} = ghostHelperUtils;
 
+// when converting data: URIs to files we need to give them unique names so
+// we don't run into server-side race condition issues with filename deduplication
+let dataImgCount = 1;
```

Comment:
I thought this had been fixed but I saw it crop up again when running locally and multiple separate image uploads were instantly triggered. Unlikely to happen in real-world conditions due to various latencies with non-local requests but something to keep an eye on.

Symptom appeared to be images being saved over the top of each other so there were duplicates showing on the page.